packages/truffle-environment/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "truffle-environment",
+  "version": "0.1.0",
+  "description": "This package contains the code needed to determine environment-specific data",
+  "main": "index.js",
```

Comment:
Might be nice to add a [basic simple README](https://github.com/trufflesuite/truffle/tree/develop/packages/truffle-blockchain-utils#readme) to the pkg and [`"repository"`](https://github.com/trufflesuite/truffle/blob/develop/packages/truffle-workflow-compile/package.json#L19), [`"bugs"/"url"`](https://github.com/trufflesuite/truffle/blob/develop/packages/truffle-workflow-compile/package.json#L22-L24), and [`"homepage"`](https://github.com/trufflesuite/truffle/blob/develop/packages/truffle-workflow-compile/package.json#L25) key/value entries to the `package.json`

Reply:
Done!